src/i18n/locale-en.json

```diff
@@ -799,6 +799,26 @@
     "query.editor.automatically.execute.update.warning": "This is an update and it may change the data in the repository.<br>Are you sure you want to execute it automatically?",
     "query.editor.error.show.full.message": "Show full exception message",
     "query.editor.error.show.less.message": "Show less exception message",
+    "yasqe.leave_page.run_queries.confirmation.none_queries_non_updates.message": "Are you sure that you want to exit?",
```

Comment:
This is not the correct place for these labels. There is the `view.sparql-editor` section where these have to go

Reply:
Done